assets/src/edit-story/components/inspector/inspectorContent.js

```diff
@@ -26,6 +26,7 @@ import { getTabId } from './utils';
 
 const InspectorPane = styled.div.attrs({ role: 'tabpanel' })`
   height: 100%;
+  overflow: overlay;
```

Comment:
This prevents shifting content in panel titles. It's only noticeable in the prepublish titles because they are the only panel titles with content against the 'edge' next to the scrollbar.